incubator/ethereum/README.md

```diff
@@ -0,0 +1,78 @@
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the vault chart and their default values.
+
+| Parameter                         | Description                                   | Default                               |
+|-----------------------------------|-----------------------------------------------|---------------------------------------|
+| `imagePullPolicy`                 | Container pull policy                         | `IfNotPresent`                        |
+| `bootnode.image.repository`       | bootnode container image to use               | `ethereum/client-go`                  |
+| `bootnode.image.tag`              | bootnode container image tag to deploy        | `alltools-v1.7.3`                     |
+| `ethstats.image.repository`       | ethstats container image to use               | `ethereumex/eth-stats-dashboard`      |
+| `ethstats.image.tag`              | ethstats container image tag to deploy        | `latest`                              |
+| `ethstats.webSocketSecret`        | ethstats secret for posting data              | `my-secret-for-connecting-to-ethstats`|
+| `ethstats.service.type`           | k8s service type for ethstats                 | `LoadBalancer`                        |
+| `geth.image.repository`           | geth container image to use                   | `ethereum/client-go`                  |
+| `geth.image.tag`                  | geth container image tag to deploy            | `v1.7.3`                              |
+| `geth.tx.replicaCount`            | geth transaction nodes replica count          | `2`                                   |
+| `geth.tx.service.type`            | k8s service type for geth transaction nodes   | `ClusterIP`                           |
+| `geth.miner.replicaCount`         | geth miner nodes replica count                | `3`                                   |
+| `geth.miner.account.secret`       | geth account secret                           | `my-secret-account-password`          |
+| `geth.genesis.networkId`          | Ethereum network id                           | `98052`                               |
+| `geth.genesis.difficulty`         | Ethereum network difficulty                   | `0x0400`                              |
+| `geth.genesis.gasLimit`           | Ethereum network gas limit                    | `0x0x8000000`                         |
+| `geth.account.publicKey`          | Geth Account to be initially funded and deposited with mined Ether | `0x975aafe74baa1978c6aabd457cf8d90ee3252db0` |
+| `geth.account.privateKey`         | Geth Private Key                              | `5c539529553e68af419860c617e20f604bd92a6f0c212b3671e0ee1312a33748` |
```

Comment:
I'd rather this is left blank - just to avoid someone accidentally starting with a well-known private key. This, the public key, all could start blank and require the user to set a value to start.

Reply:
Public key is required for the miners as it needs a default account to deposit ether. If this is empty, miner's won't come up.

> just to avoid someone accidentally starting with a well-known private key
To re-iterate, this is a *private* Ethereum network. This network has a *different* genesis block than mainnet so there's no way it can connect to the public network.

Reply:
I understand that it's private. It's not a security question, it's just an uncommon pattern to do this I think. 

cc/ @unguiculus who does a lot more chart reviews

Reply:
FWIW, for the burrow chart we had to do adopt a similar pattern, or else the tests wouldn't pass...